src/en/authors-charm-store.md

```diff
@@ -31,6 +36,11 @@ following information:
  - `Two-factor auth` - If enabled, enter two-factor authentication (2FA)
 information. Otherwise <kbd>Return</kbd> for None
 
+Also note, if your group memberships change on Launchpad, you'll need to
```

Comment:
Seems like an Oxford comma ("if" is conjunctive enough, right?) but probably should `s/Also note,/Also note:/`.